src/store/sagas/getForecastAuto.js

```diff
@@ -1,14 +1,18 @@
 import {
   call, put, takeEvery,
 } from 'redux-saga/effects';
-import axios from 'axios';
 import * as actions from '../constants';
 
 function fetchForecastAuto({ name, key }) {
-  return axios
-    .get(`https://api.weatherbit.io/v2.0/forecast/daily?city=${name}&key=${key}`)
-    .then((res) => res.data)
-    .catch((err) => (err.response ? err.response.data.message : err.message));
+  return fetch(`https://api.weatherbit.io/v2.0/forecast/daily?city=${name}&key=${key}`)
+    .then((response) => {
+      if (response.status === 200) {
+        return response.json();
+      }
+      return null;
```

Comment:
We can do `throw new Error` here so that it got caught in catch and then do something with it in catch